qemu/tests/cfg/usb_dev.cfg

```diff
@@ -0,0 +1,78 @@
+- usb_dev:
+    virt_test_type = qemu
+    no Host_RHEL.m6.u1
+    restart_vm = yes
+    kill_vm_on_error = yes
+    usbs = " usbtest"
+    usb_bus = "usbtest.0"
+
+    # usb controllers
+    variants:
+        - piix3-uhci:
+            usb_type_usbtest = piix3-usb-uhci
+            usb_controller = uhci
+            usb_max_port_usbtest = 2
+            drive_format_stg = "usb1"
+            no ppc64 ppc64le
+        - piix4-uhci:
+            usb_type_usbtest = piix4-usb-uhci
+            usb_controller = uhci
+            usb_max_port_usbtest = 2
+            drive_format_stg = "usb1"
+            no ppc64 ppc64le
+        - ich9-uhci:
+            usb_type_usbtest = ich9-usb-uhci6
+            usb_controller = uhci
+            usb_max_port_usbtest = 2
+            drive_format_stg = "usb1"
+            no ppc64 ppc64le
+            no Host_RHEL.m6
+        - ich9-ehci:
+            usb_type_usbtest = ich9-usb-ehci1
+            usb_controller = ehci
+            usb_max_port_usbtest = 6
+            drive_format_stg = "usb2"
+        - usb-ehci:
+            usb_type_usbtest = usb-ehci
+            usb_controller = ehci
+            usb_max_port_usbtest = 6
+            drive_format_stg = "usb2"
+        - nec-xhci:
+            no RHEL.3
+            no RHEL.4
+            no RHEL.5
+            no Host_RHEL.m6
+            usb_type_usbtest = nec-usb-xhci
+            usb_controller = xhci
+            usb_max_port_usbtest = 4
+            drive_format_stg = "usb3"
+            Host_RHEL:
+                no Win2000, WinXP, Win2003, WinVista, Win7, Win2008
+
+    # usb toplogy
+    variants:
+        - without_usb_hub:
+
+    # usb tests
+    variants:
+        - usb_stg_hotplug:
+            type = usb_block_dev_hotplug
+            images += " stg"
+            image_name_stg = "images/usbdevice"
+            image_format_stg = "qcow2"
+            image_boot_stg = no
+            create_image_stg = yes
+            remove_image_stg = yes
+            image_size_stg = 1G
+            blk_extra_params_stg = "serial=TARGET_DISK0" 
+            chk_usb_info_cmd = "lsusb -v"
+            Windows:
+                chk_usb_info_cmd = 'wmic path Win32_USBControllerDevice get Dependent | find "USB"'
+            device_type = "usb-storage"
+            product = "Product QEMU USB MSD"
+            cmd_timeout = 180
+            variants:               
+                - one_time:
+                    repeat_times = 1
+                - multi_times:
+                    repeat_times = 1000
```

Comment:
For the scenario 'Hot plug and unplug usb storage device 1000 times', the steps should be repeat(hotplug-> unplug), instead of repeat(unplug->hotplug), which are different.